query-connector/src/app/shared/page-routes.ts

```diff
@@ -1,44 +1,32 @@
-import { RoleTypeValues } from "../models/entities/user-management";
+import { UserRole } from "../models/entities/user-management";
 
 /**
  * Pathnames
  */
 export enum PAGES {
   LANDING = "/",
   QUERY = "/query",
-  MY_QUERIES = "/queryBuilding",
+  QUERY_BUILDING = "/queryBuilding",
```

Comment:
keeping names mapped to component/path names

Reply:
It doesn't need to happen in this PR but could we rename the path `/queryBuilding` to `/myQueries`. This seems to be the only path that does not math its name in the menu 🤔 .

Reply:
Feel free to resolve this convo so it unblocks merging.